subprojects/plugins/src/integTest/groovy/org/gradle/groovy/compile/BasicGroovyCompilerIntegrationSpec.groovy

```diff
@@ -122,6 +136,7 @@ abstract class BasicGroovyCompilerIntegrationSpec extends MultiVersionIntegratio
         !groovyClassFile('Groovy$$Generated.java').exists()
         !groovyClassFile('Groovy.class').exists()
         !groovyClassFile('Groovy$$Generated.class').exists()
+        compileWithAnnotationProcessingOperation(':compileGroovy').failure.contains("Compilation failed")
```

Comment:
I feel like asserting this once for the simplest failure case should be enough. Beyond that it doesn't really provide additional safety and makes the tests more verbose.

Reply:
Agreed and removed in 0819ec34f23.